### ISSUE TITLE
horizontal_figure input grid order

### ISSUE THREAD

**Issue by PearCoding** [Mira-13/figure-gen#1]
It seems the order of the input grids in `horizontal_figure` has to be swapped and is inconsistent to the provided [pool.py](https://github.com/Mira-13/figure-gen/blob/b127e368899ed3e31cd412e0faee5b5c5a130cb1/tests/pool.py#L96) test case.

E.g: (before)
```python
figuregen.horizontal_figure([ref_grid, comp_grid], width_cm=15., filename='siggraph/'+scene+'.pdf')
```

(after)
```python
figuregen.horizontal_figure([comp_grid, ref_grid], width_cm=15., filename='siggraph/'+scene+'.pdf')
```

**Comment by PearCoding** [comment]
To be precise: The problem only appears while using the LaTex backend, html is producing the expected result. pptx was not tested.

**Comment by pgrit** [comment]
Are you using the latest version? This should be fixed by bf848de4ed55108ead004d4fc217139f5b2373fb and better handled by b127e368899ed3e31cd412e0faee5b5c5a130cb1

**Comment by PearCoding** [comment]
With the recent version it works.
However, the line in [setup.sh](https://github.com/Mira-13/figure-gen/blob/b127e368899ed3e31cd412e0faee5b5c5a130cb1/setup.sh#L4) has to change from
```bash
python -m pip install --upgrade .\dist\figuregen-0.2.1-py3-none-any.whl
```
to
```bash
python -m pip install --upgrade ./dist/figuregen-0.2.2-py3-none-any.whl
```
to work within the linux environment. The backslash is not valid in bash and the python package has the version 0.2.2

**Comment by pgrit** [comment]
Thanks for checking! Fixed in 69f84ae1f77c4173224a6d89c39f3c7f1decac91
